meta/store.go

```diff
@@ -1039,6 +1039,29 @@ func (s *Store) CreateDatabase(name string) (*DatabaseInfo, error) {
 	return s.Database(name)
 }
 
+// CreateDatabaseWithRetentionPolicy creates a new database with an explicit retention policy in the store.
+func (s *Store) CreateDatabaseWithRetentionPolicy(name string, rpi *RetentionPolicyInfo) (*DatabaseInfo, error) {
+	if err := s.exec(internal.Command_CreateDatabaseCommand, internal.E_CreateDatabaseCommand_Command,
```

Comment:
Is this deliberate? Call `exec` to create the database, but use the function to create the retention policy? I guess it is, due to the auto-create functionality in `CreateDatabase`.